cartography/intel/github/repos.py

```diff
@@ -292,6 +357,30 @@ def load_github_owners(session, update_tag, repo_owners):
         )
 
 
+@timeit
+def load_python_requirements(neo4j_session, update_tag, requirements_objects):
+    query = """
+    UNWIND {Requirements} AS req
+        MERGE (lib:PythonLibrary:Dependency{id: req.id})
+        ON CREATE SET lib.firstseen = timestamp(),
+        lib.name = req.name
+        SET lib.version = req.version,
+        lib.uri = req.uri,
+        lib.lastupdated = {UpdateTag}
+
+        WITH lib, req
+        MATCH (repo:GitHubRepository{id: req.repo_url})
```

Comment:
Please give context on why MATCH is used here vs MERGE. It's unclear to me where the node would be created in the sync.

Reply:
> It's unclear to me where the node would be created in the sync.

I moved out all the `load*` functions into a function called `load()` below to hopefully make this more clear. We call `load_github_repos()` first to load repos so by the time `load_python_requirements()` is called, the repo data is already in the graph and we can just match on them.